src/physics/navier_stokes.cpp

```diff
@@ -365,8 +625,9 @@ ::compute_deviatoric_strain_rate_tensor (
     dealii::Tensor<2,dim,real> deviatoric_strain_rate_tensor;
     for(int d1=0; d1<dim; ++d1) {
         for(int d2=0; d2<dim; ++d2) {
-            deviatoric_strain_rate_tensor[d1][d2] = strain_rate_tensor[d1][d2] - (1.0/3.0)*vel_divergence;
+            deviatoric_strain_rate_tensor[d1][d2] = strain_rate_tensor[d1][d2];
         }
+        deviatoric_strain_rate_tensor[d1][d1] -= (1.0/3.0)*vel_divergence;
```

Comment:
This looks different from before. Are there any impacts on results using NS without this change?

Reply:
It only corrects a specific post processed quantity in the TGV case, so no it wont impact anyone's results -- it corrects the viscous dissipation rate in the KE budget which no one else in the lab is looking at

Reply:
Sounds good. Thanks for clarifying!